modules/serverless-ossm-setup.adoc

```diff
@@ -183,6 +184,147 @@ spec:
 $ oc apply -f <filename>
 ----
 
+. Install Knative Eventing by creating the following `KnativeEventing` custom resource definition (CRD), which also enables the Istio integration:
```

Comment:
maybe we can have this (and serving, and knative kafka), as sub items on the Navigation bar in the documentation?